### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - Florian Chollet - Alexandre Delétraz

### DISCUSSION THREAD

**Discussion by luma2010** [heig-vd-dai-course/heig-vd-dai-course#548] [Show and tell]
Notre idée pour le travail pratique 3 est de faire un site qui gère des notes. Sur ce site nous pouvons ajouter des cours (BDR, DAI, PRG1,etc...), puis rajouter des notes à un de ces cours. Il sera ensuite possible de consulter la moyenne du cours via une page de résumer.

Comme objectif optionnels si nous avons le temps  :
- Ajouter une gestions d'utilisateurs avec des cookies
- Ajouter un système de pondération
- Ajouter un système de classe (calcule de moyenne de la classe)

**Comment by ludelafo** [comment]
OK pour nous !

**Comment by luma2010** [comment]
Voici notre projet pratique 3 avec le dernier commit :
https://github.com/luma2010/heig-vd-dai-projet-pratique3.git

Dernier commit : 807ec062e6b6e5cfe5e9c317f2fc6d38a26b66fa
